### PR TITLE
issue #229 Room creation with invalid slug doesnt error out.

### PR DESCRIPTION
**Pull request by rjschu**
hey hope this does what you where after, if not id happily take another shot at it :) ive attached an image of the error that is shown also 
screenshot from 2015-02-14 17 42 06

### CODE REVIEW

File: media/js/client.js [hunk 1]
```diff
@@ -43,7 +43,12 @@
         };
         var callback = data.callback;
         this.socket.emit('rooms:create', room, function(room) {
-            if (room && room.id) {
+            if (room && room.errors) {
+                swal("Unable to create room",
+                     "Room slugs can only contain lower case letters, numbers or underscores!",
+                     "error");
+            }
```
Comment (hhaidar) [inline media/js/client.js]: brace and else should be on the same line `} else..`

**Comment by hhaidar** [comment]
We gotta make it spit out real errors (like uniqueness of slug, etc), but this looks good for now – just one minor style thing.

**Comment by rjschu** [comment]
by real errors do you mean the error thats being returned in the room object? or a message based on what the error message is?

**Comment by hhaidar** [comment]
@rjschu both lol. We'll need to normalize the errors being returned from the server and then display them properly on all forms and stuff.

Anyway, let's merge this in and I'll file a separate issue for the errors stuff.

**Comment by hhaidar** [comment]
@rjschu thanks!

**Comment by rjschu** [comment]
@hhaidar no worries :)

**Pull request merged**